honkbot.py

```diff
@@ -1,23 +1,23 @@
+# Python Standard Library
 import datetime
-import discord
+import inspect
 import logging
 import os
 import pytz
-import sys
-from bs4 import BeautifulSoup
 import requests
-from typing import Optional
+import sys
 
+# Packages
+import discord
 from discord.errors import Forbidden
+from discord.ext import commands
 import dotenv
 
-logging.basicConfig(stream=sys.stdout, level=logging.WARN)
-logger = logging.getLogger(__name__)
-logger.setLevel(logging.INFO)
-logger.info("Starting Honkbot...")
+# Honkbot
+import remy
 
```

Comment:
I alphabetized and categorized these. I also moved the stuff at the top down into the main function at the bottom.